src/styles/layout/_portfolio.scss

```diff
@@ -0,0 +1,48 @@
+// Styling for the portfolio grid section
```

Comment:
All of your styles associated with a component need to be nested relative to their corresponding component. Right now it's very hard to find the SCSS files. If an SCSS file is global that's fine to keep in styles. But not for component related files.